addons/l10n_vn/data/account.account.template.csv

```diff
@@ -28,7 +28,7 @@
 "chart1533","Instruments for renting",1533,"asset_current","vn_template","False"
 "chart1534","Equipment and spare parts for replacement",1534,"asset_current","vn_template","False"
 "chart1541","Construction contracts",1541,"asset_current","vn_template","False"
-"chart1542","Other products",1542,"asset_current","vn_template","False"
+"chart1542","Other Work-in-progress products",1542,"asset_current","vn_template","False"
```

Comment:
```suggestion
"chart1542","Other work-in-progress products",1542,"asset_current","vn_template","False"
```

Reply:
@lordkrandel tks for your review. I dit it according to your comment, and rebased too. Please help review again.